projects/vdk-plugins/vdk-oracle/README.md

```diff
@@ -199,6 +199,81 @@ def run(job_input):
     job_input.send_object_for_ingestion(payload=payload, destination_table="test_table")
 ```
 
+### Multiple Oracle Database
+
+#### Configuring Multiple Oracle Databases
+
+To manage multiple Oracle database connections within a data job,
+always configure the default database in the `[vdk]` section of the `config.ini` file.
+This section should contain the primary connection details that the application will use by default.
+
+For each additional Oracle database,
+add a new section following the pattern `vdk_<name>`, where `<name>` is a unique identifier for each database connection.
+These additional sections must also include all necessary Oracle connection details.
+
+#### Example `config.ini` with Multiple Oracle Databases
+
+```ini
+[vdk]
+oracle_user=user
+oracle_password=password
+oracle_host=localhost
+oracle_port=1521
+oracle_sid=FREE
+oracle_connection_string =localhost:1521/FREE
+oracle_thick_mode=True
+
+[vdk_oracle_reports]
+oracle_user=reports_user
+oracle_password=reports_password
+oracle_host=localhost
+oracle_port=1523
+oracle_sid=FREE
+oracle_connection_string =localhost:1523/FREE
+oracle_thick_mode=False
+
+```
+
+You can specify which database to use in your data job by referencing the specific section name.
```

Comment:
Does it make sense to mention that  the section name, is going to be lowercased and used as the DB connection name?

Reply:
I added a note in the readme